src/cclib/bridge/cclib2openbabel.py

```diff
@@ -40,6 +40,35 @@ def makeopenbabel(atomcoords, atomnos, charge=0, mult=1):
     obmol.SetTotalCharge(charge)
     return obmol
 
+def makecclib(mol):
+    """Create cclib attributes from an OpenBabel molecule.
+
+    Beyond the numbers, masses and coordinates, we could also set the total charge
+    and multiplicity, but often these are calculated from atomic formal charges
+    so it is better to assume that would not be correct.
+    """
+
+    attributes = {
+        'atomcoords':   [],
+        'atommasses':   [],
+        'atomnos':      [],
+        'natom':        mol.NumAtoms(),
+    }
+    for atom in ob.OBMolAtomIter(mol):
+        attributes['atomcoords'].append([atom.GetX(), atom.GetY(), atom.GetZ()])
+        attributes['atommasses'].append(atom.GetAtomicMass())
+        attributes['atomnos'].append(atom.GetAtomicNum())
+    return attributes
+
+def readfile(fname, type):
+    """Read a file with OpenBabel and extract cclib attributes."""
+
+    obc = ob.OBConversion()
+    obc.SetInFormat(type)
```

Comment:
Should we check that the format library is correctly loaded and if not, log a warning?

```
retval = obc.SetInFormat(type)
if not retval:
   logging.warning("Unable to load the %s reader from OpenBabel.")
```

Also, should we ensure that ```type``` is a string?